addons/binding/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/welcome/NAWelcomePersonHandler.java

```diff
@@ -0,0 +1,120 @@
+                    && (lastEvent == null || lastEvent.getTime() < event.getTime())) {
+                lastEvent = event;
+            }
+        });
+
+        super.updateChannels(module);
+    }
+
+    @Override
+    protected State getNAThingProperty(String channelId) {
+        if (module != null) {
+            switch (channelId) {
+                case CHANNEL_WELCOME_PERSON_LASTSEEN:
+                    return toDateTimeType(module.getLastSeen());
+                case CHANNEL_WELCOME_PERSON_ATHOME:
+                    return module.getOutOfSight() ? OnOffType.OFF : OnOffType.ON;
+                case CHANNEL_WELCOME_PERSON_AVATAR_URL:
+                    return toStringType(getAvatarURL());
+                case CHANNEL_WELCOME_PERSON_AVATAR:
+                    return getRawDataFromPictureUrl(getAvatarURL());
+                case CHANNEL_WELCOME_PERSON_LASTMESSAGE:
+                    if (lastEvent != null && lastEvent.getMessage() != null) {
+                        return toStringType(lastEvent.getMessage().replace("<b>", "").replace("</b>", ""));
+                    }
+                    return UnDefType.UNDEF;
+                case CHANNEL_WELCOME_PERSON_LASTTIME:
+                    return lastEvent != null ? toDateTimeType(lastEvent.getTime()) : UnDefType.UNDEF;
+                case CHANNEL_WELCOME_PERSON_LASTEVENT:
+                    return getLastEventURL() != null ? getRawDataFromPictureUrl(getLastEventURL()) :
+
+                            UnDefType.UNDEF;
+                case CHANNEL_WELCOME_PERSON_LASTEVENT_URL:
+                    return toStringType(getLastEventURL());
+            }
+        }
+        return super.getNAThingProperty(channelId);
+    }
+
+    private String getLastEventURL() {
+        if (lastEvent != null && lastEvent.getSnapshot() != null) {
+            return getBridgeHandler().getPictureUrl(lastEvent.getSnapshot().getId(), lastEvent.getSnapshot().getKey());
+        }
+        return null;
+    }
+
+    private String getAvatarURL() {
+        if (avatarURL == null && module.getFace() != null) {
+            avatarURL = getBridgeHandler().getPictureUrl(module.getFace().getId(), module.getFace().getKey());
+        }
+        return avatarURL;
+    }
+
+    @Override
+    public void handleCommand(ChannelUID channelUID, Command command) {
+        super.handleCommand(channelUID, command);
+        if ((command instanceof OnOffType) && (CHANNEL_WELCOME_PERSON_ATHOME.equalsIgnoreCase(channelUID.getId()))) {
+            if ((OnOffType) command == OnOffType.OFF) {
+                getBridgeHandler().getWelcomeApi().setpersonsaway(getParentId(), getId());
+            } else {
+                // Experimental, this method is not documented in the API but **seems** to work
+                // Playing to much with it seems to lead to connection refused
+                getBridgeHandler().getWelcomeApi().setpersonshome(getParentId(), "[\"" + getId() + "\"]");
```

Comment:
I tested and It seems to not work.

Reply:
Really not sure. Have you tested updating in OH2 toward what is presented in my.netatmo.com ?

Reply:
Just read your comment below : I agree : 
- switching person away works but switching it at home does not. As this is not officially documented in netatmo API, I think I would have better document it in README and maybe remove this piece of code